styles.css

```diff
@@ -60,6 +60,38 @@ body {
   #directorTextDiv {
     padding: 0rem 2rem 0rem 2rem;
   }
+
+  #servicesHeading {
+    font-size: 3.5rem;
+  }
+
+  #servicesTextRow {
+    margin-top: -2.5rem;
+  }
+
+  #historyHeading {
+    text-align: left;
+  }
+
+  .historyLeftCol {
+    border-left: 1px solid #e6e6e6; 
+    padding: 0.5rem 0rem 0.5rem 1.5rem;
+    margin: 0rem 0rem 0rem 2rem;
```

Comment:
can just use `margin-left: 2rem` since you're setting margin for the bottom (unless you are intentionally overriding margin top, right and bottom to 0)